<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrnGenerateColEntryWidget</class>
 <widget class="QWidget">
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="entryHorizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="LineEditWidget" name="firstColorLineEdit">
       <property name="charWidth">
        <number>4</number>
       </property>
       <property name="toolTip">
        <string>Index of the first color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEditWidget" name="lastColorLineEdit">
       <property name="charWidth">
        <number>4</number>
       </property>
       <property name="toolTip">
        <string>Index of the last color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="shadeColorCheckBox">
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>These colors must not change</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
